health-services/project-factory/src/server/utils/campaignUtils.ts

```
@@ -2385,6 +2507,40 @@ async function createProject(
   );
 }
 
+const enrichTargetForProject = (project: any, codesTargetMapping: any, boundaryCode: any) => {
+  if (codesTargetMapping?.[boundaryCode] && Object.keys(codesTargetMapping[boundaryCode]).length > 0) {
+    let targets: any[] = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'targets' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)